Include/Ashkal/Shapes/Stage.hpp

```diff
@@ -15,7 +16,7 @@ namespace Ashkal {
 
       //! Returns the vector of shapes and their positions.
       const std::vector<std::pair<std::unique_ptr<Shape>, Point>>&
```

Comment:
Remove the ```Point``` since it's no longer needed. A ```Stage``` is now a vector of ```Shape```s.

Reply:
Are they still unique_ptrs?

Reply:
Determine the best solution.